pom.xml

```
@@ -43,11 +43,14 @@
         <mule.wss.module.version>1.2.2</mule.wss.module.version>
         <easy-random-core>4.3.0</easy-random-core>
 
+        <muleSchedulerServiceVersion>1.9.4</muleSchedulerServiceVersion>
+        <muleHttpServiceVersion>1.11.4</muleHttpServiceVersion>
+
         <!-- Remove when a new parent version with MTF is available -->
         <!-- MTF dependencies -->
         <mtf.tools.version>1.2.0</mtf.tools.version>
         <munit.extensions.maven.plugin.version>1.5.0</munit.extensions.maven.plugin.version>
-        <minVersion>4.3.0</minVersion>
+        <minVersion>4.9.0</minVersion>
```

```suggestion
        <minTestingVersion>4.9.0</minTestingVersion>
```

Consider renaming the property so it is never used for other purposes. The actual MMV of the connector should be defined as the mule api version you integrate and it is defined in the parent hierarchy.

---

Updated